docs/Troubleshooting.md

```diff
@@ -138,3 +138,99 @@ and is a JSON object generated by the Quilt build system.
 
     `$ aws cloudformation describe-stack-events --stack-name QUILT_STACK_NAME`
 
+## Collecting logs
+
+To expedite the resolution of any errors encountered while using
+Quilt, please capture the following logs and attach to an email
+to support@quiltdata.io:
+
+1. Browser logs (isolating the most recent request that triggered an error):
+    1. Go to the affected page in your Quilt Catalog.
+    1. Open the browser Developer tools:
+        - Google Chrome: Press **F12**, **Ctrl+Shift+I** or from the
+        Chrome menu select **More tools > Developer tools**.
+        - Mozilla Firefox: Press **F12** or go to **Tools > Browser Tools > Web Developer Tools**.
+        - Apple Safari: Go to **Develop > Show Web Inspector**.
+    1. Select the **Network** tab.
+    1. Ensure the session is recorded:
+        - Google Chrome: Check the red button in the upper left corner is set to **Record**.
+        - Mozilla Firefox: Check the Pause/Resume button is set to **Resume**.
+        - Apple Safari: Sessions are recorded by default.
+    1. Ensure **Preserve Log** is enabled (**Persist Logs** in Mozilla Firefox).
+    1. Click the **Clear** button to clear out existing logs.
+    1. Perform the action that triggers the error (e.g. clicking the `Download package` button).
+    1. Export the logs as HAR format.
+        - Google Chrome: **Ctrl + Click** anywhere on the grid of
+        network requests and select **Save all as HAR with content**.
+        - Mozilla Firefox: Select the gear icon () and select **Save
+        All As HAR**.
+        - Apple Safari: Click the **Export** button
+    1. Save the HAR-formatted file to your localhost.
+        ![Save browser Network error logs as HAR content](imgs/troubleshooting-logs-browser.png)
+1. CloudWatch logs for the Quilt cluster Elastic Container Service:
+    1. Install the [AWS CLI](https://aws.amazon.com/cli/) tool.
+    1. Export the AWS profile for your Quilt stack:
+        ```
+        export AWS_PROFILE=stage
+        ```
+    1. Execute the `cloudformation` command to retrieve your Quilt stack log group name:
```

Comment:
`cloudformation` isn't the name of the command. Also let's simplify this whole thing and ask the user to first get the name of the affected  Quilt CF stack (`aws cloudformation list-stacks` is a starting point).

And because in the CFT we do `LogGroupName: !Ref 'AWS::StackName'`, for now we can simply assume that the name of the LogGroup IS the stack name and the user can proceed from there.